codegen/HackCw20.message-composer.ts

```diff
@@ -0,0 +1,395 @@
+/**
```

Comment:
Why are you using the message-composers everywhere?

You can just use the SigningClients.

Message Composers are great for when you want to eg. prepare some message for a DAO to execute. But let's just execute them now (well, send them to keplr, where it asks for use approval)